security/acls/acl.sh

```diff
@@ -168,8 +166,8 @@ ccloud service-account create $SERVICE_NAME --description $SERVICE_NAME || true
 SERVICE_ACCOUNT_ID=$(ccloud service-account list | grep $SERVICE_NAME | awk '{print $1;}')
 
 echo -e "\n# Create an API key and secret for the new service account"
-echo "ccloud api-key create --service-account-id $SERVICE_ACCOUNT_ID --cluster $CLUSTER"
-OUTPUT=$(ccloud api-key create --service-account-id $SERVICE_ACCOUNT_ID --cluster $CLUSTER)
+echo "ccloud api-key create --service-account-id $SERVICE_ACCOUNT_ID --resource $CLUSTER"
```

Comment:
@rspurgeon for consideration, either:

1. Leave this to a different PR (DEVX-1084)
2. Update the script to check that the CLI version accepts `--resource` flag (requires enhancement to the `check_ccloud_v2` function)

Reply:
preferred approach?

Reply:
Actually, my vote is BOTH (not either/or) for clarity and traceability.

So specifically, the suggestion would be:

1. In this PR, remove the change from `--cluster` to `--resource`.  Restore to `--cluster`
2. Submit a separate PR for the change from `--cluster` to `--resource`.
3. In that separate PR, augment `check_ccloud_v2` function to check the CLI version number...which will be useful down the road as well, to accommodate new CLI features and/or breaking changes